app/code/Magento/Sales/view/frontend/templates/order/print/shipment.phtml

```diff
@@ -3,26 +3,34 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
+use Magento\Framework\Escaper;
```

Comment:
We don't use imports in `phtml` files.

Reply:
Do we have such rule? It seems to me quite stupid. Don't you thing so?

Reply:
I'm not sure if `stupid` is an appropriate word for what, however:
- Importing class for single annotation is - in my opinion - not necessary
- we should keep consistency.
- PHTML file is not a regular PHP file, as it's passed through the Template Engine, thus - introducing elements such as `use` might cause issues 

I'll try to find resource about this for you @ihor-sviziev